tests/unit/src/main/scala/tests/BaseCodeLensLspSuite.scala

```diff
@@ -10,13 +10,14 @@ class BaseCodeLensLspSuite(name: String) extends BaseLspSuite(name) {
   def check(
       name: TestOptions,
       library: Option[String] = None,
-      scalaVersion: Option[String] = None
+      scalaVersion: Option[String] = None,
+      printCommand: Boolean = false
   )(
       expected: String
   )(implicit loc: Location): Unit = {
     test(name) {
       cleanWorkspace()
-      val original = expected.replaceAll("<<.*>>[^a-zA-Z0-9@]+", "")
+      val original = expected.replaceAll("<<.*>>[^a-zA-Z0-9@]+$", "")
```

Comment:
I was wondering why [this test](https://github.com/scalameta/metals/pull/3154/files#r716428172) passes on and found that input isn't cleaned from lenses because of wrong regex.

It should be:
```suggestion
      val original = expected.replaceAll(s"<<.*>>${System.lineSeparator()}", "")
```

Reply:
`$` is already used for end of line, so this should be fine, no?

Reply:
Ach, actually it is no, which is super weird.

Reply:
Ach, silly me `$` matches end of string not end of line. Should be fixed now!